app/controllers/risc/configuration_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Risc
+  class ConfigurationController < ApplicationController
+    def index
+      render json: RiscConfigurationPresenter.new.configuration
```

Comment:
When I added the OpenID discovery endpoint, I added caching headers because I'm pretty sure some specification required it....but I can't find it and didn't see any caching direction on the specs for this. I left out caching, but it's a one-liner so happy to add it if anybody feels strongly